spec/system/index_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'spec_helper'
+
+describe 'searching for a bus stop by stop id' do
+  let(:user) { create :user }
+  let!(:bus_stop) { create :bus_stop }
+  before :each do
+    when_current_user_is user
+    visit root_url
+  end
+  context 'correct stop id' do
+    it 'redirects to the edit page' do
+      within 'form', text: 'Enter stop ID' do
+        fill_in 'Enter stop ID', with: bus_stop.hastus_id.to_s
+        click_button 'Search'
+      end
+      expect(page).to have_content "Editing #{bus_stop.name}"
```

Comment:
I'm unfamiliar with this app so I'm curious. Does clicking search only have the options of bringing up the edit page for a specific stop or telling you the stop doesn't exist? Usually searches give you a list of possible options to select from. This is less of a "search" and more of a "find" I guess.